.github/workflows/selfdrive_tests.yaml

```diff
@@ -227,7 +227,7 @@ jobs:
     - name: Build openpilot
       run: ${{ env.RUN }} "scons -j$(nproc)"
     - name: Test car models
-      timeout-minutes: ${{ contains(runner.name, 'nsc') && (steps.routes-cache.outputs.cache-hit == 'true') && 1 || 20 }}
+      timeout-minutes: ${{ contains(runner.name, 'nsc') && (steps.routes-cache.outputs.cache-hit == 'true') && 1 || 3 }}
```

Comment:
We might want to keep it above 5 minutes, since this is also the timeout for non namespace runner. This PR form deanlee takes more than 4 minutes to run the test: https://github.com/commaai/openpilot/actions/runs/13748851002/job/38447203476?pr=34825